### PR TITLE
chore: update i18n change

### PR DESCRIPTION
**Pull request by tunv8zzz**
## Please verify the following:

- [x] `yarn test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR
- Language editing demo.
- i think a minimal app will also have a language switch function in the app.

### CODE REVIEW

**Comment by frankcalise** [comment]
Hi @luvyouso, would you mind editing the PR comment and writing up a description for your PR as to why it is needed and what it does?

Thank you for your contribution!

**Comment by luvyouso** [comment]
> Hi @luvyouso, would you mind editing the PR comment and writing up a description for your PR as to why it is needed and what it does?
> 
> Thank you for your contribution!

hi @frankcalise , i update PR.

File: boilerplate/app/i18n/LocaleContext.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import React from "react"
+import I18n from "i18n-js"
+import { translate } from "./translate"
+import { useStorage } from "../utils/storage"
+import { UPDATE_LANGUAGE } from "../utils/constants"
+
+const LocaleContext = React.createContext<any>(null)
+
+export const LocaleContextProvider = (props) => {
+  const [locale, changeLocale] = useStorage(UPDATE_LANGUAGE, "en")
```
Comment (sergioisidoro) [inline boilerplate/app/i18n/LocaleContext.tsx:11]: Does this mean we lose the default usage of the system language, and hardcode english?

I just made a proposal tangentially related to this:
https://github.com/infinitered/ignite/pull/2415

**Comment by ahmedrowaihi** [comment]
### Update: 
I updated the [example](https://github.com/ahmedrowaihi/react-native-i18n-storage/tree/main/example)
I will update the instructions once I finish the IOS as well.

for now you can just install packages and run example, and start toggling the locale, it will restart the app, also if you fully close the app and re-run it agian, you will see that the latest locale is synched from the locale storage successfully, and it's working like charm! with no hooks required, no second render, no Async on JS.

**Comment by tunv8zzz** [comment]
> ### Update:
> I updated the [example](https://github.com/ahmedrowaihi/react-native-i18n-storage/tree/main/example) I will update the instructions once I finish the IOS as well.
> 
> for now you can just install packages and run example, and start toggling the locale, it will restart the app, also if you fully close the app and re-run it agian, you will see that the latest locale is synched from the locale storage successfully, and it's working like charm! with no hooks required, no second render, no Async on JS.

thanks for your support. @ahmedrowaihi

**Comment by ahmedrowaihi** [comment]
> 

my pleasure! @tunv8zzz 

I just released an initial PoF, with Docs on how to use it.
[v1.2.0](https://github.com/ahmedrowaihi/react-native-i18n-storage/releases/tag/v1.2.0)

**Comment by frankcalise** [comment]
Thanks for the contribution but we are not looking to pull this into Ignite at this time. I believe if we changed languages from LTR to RTL it wouldn't be reflected still.

**Pull request closed without merging**